content/docs/api-reference/make_checkpoint.md

```diff
@@ -20,9 +20,9 @@ while True:
 ## Description
 
 To track successive steps in a longer <abbr>experiment</abbr>, you can write
-your code so it registers checkpoints with DVC during runtime. This function
-should be called by the code in stages executes by `dvc exp run` (see `cmd`
-field of `dvc.yaml`).
+your code so it registers checkpoints with DVC during runtime (similar to a
+logger). This function can be called by the code in stages executes by
+`dvc exp run` (see also the `cmd` field of `dvc.yaml`).
```

Comment:
Is this supposed to refer to the `checkpoint` subfield of `output`? I'm unclear how `cmd` is related. Does it make sense to include an example `dvc.yaml`?

Reply:
No, I meant to mention `cmd` but yeah it's not clear, thanks. Updated